### ISSUE TITLE
Cannot update due to server version 24 or lower being required

### ISSUE THREAD

**Issue by SnorrenEnzo** [julien-nc/cospend-nc#208]
At the Nextcloud update to 25.0, after not being able to cancel the updating process, I suddenly got a message that there was no version of Cospend available for that version of Nextcloud. Since then, I have not been able to update Cospend to the newest version, always getting the below message. Is there anything I can do to resolve this? Cospend is version 1.4.8, Nextcloud version 25.0.5.

![screenshot](https://github.com/julien-nc/cospend-nc/assets/31887537/7fa90c92-42ce-4568-82e9-fbf6f4fe0721)

**Comment by ThisIsMe-TIM** [comment]
strange ... My nextcloud is 26.0.1 and I just apply (few minutes ago ;) ) the new version of cospend : 1.5.10 
May be uninstall cospend and install again ?

**Comment by SnorrenEnzo** [comment]
Yes I thought of that too. Does this lead to a loss of data though? I haven't found a way to backup the Cospend projects except through a total copy of Nextcloud.

**Comment by ThisIsMe-TIM** [comment]
to be sure of not losing data, you can export all projects one by one (project/params ): it gives you a csv file into a Cospend directory

**Comment by SnorrenEnzo** [comment]
Last time I did this it did not properly import all the categories. However, I guess I don't have any other option so I will try that.

**Comment by SnorrenEnzo** [comment]
It turns out I seem to be unable to uninstall Cospend. Removing via the UI results in the following:

![Screenshot from 2023-05-27 18-17-05](https://github.com/julien-nc/cospend-nc/assets/31887537/bbe2c8ee-760a-454c-a2e0-c1375c9b2b24)

Uninstalling and reinstalling via the command line results in the same paradox:

`# sudo -u #80 php ./occ app:remove cospend
cospend disabled
cospend 1.4.3 removed
# sudo -u #80 php ./occ app:install cospend
cospend already installed`

I run Nextcloud inside a TrueNAS jail. Rebooting the jail yields the same result, not being able to uninstall Cospend fully. How do I go about actually removing it for a fresh install of Cospend? Can I remove an installation directory?

**Comment by SnorrenEnzo** [comment]
Anyone have any suggestions on how to properly remove Cospend and reinstall it? Searches on Google regarding this turn out blank.
